generator/src/test/java/com/scottlogic/deg/generator/outputs/targets/FileOutputTargetTests.java

```diff
@@ -0,0 +1,84 @@
+package com.scottlogic.deg.generator.outputs.targets;
+
+import com.scottlogic.deg.generator.outputs.formats.OutputFormat;
+import com.scottlogic.deg.generator.utils.FileUtils;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class FileOutputTargetTests {
+    @Mock
+    private Path mockFilePath;
+    @Mock
+    private FileUtils mockFileUtils;
+    @Mock
+    private OutputFormat mockOutputFormat;
+    @Mock
+    private Path mockParentPath;
+
+    @Test
+    public void validate_generateOutputFileIsADirectory_throwsException(){
```

Comment:
Picky observation: on this method you have written `(){` with no spaces.  On the one below you have `() {`.  I prefer the latter but it would be nice to see consistency.